_rules/SC4-1-2-aria-attr-valid.md

```diff
@@ -0,0 +1,118 @@
+---
+name: Validity of ARIA attribute
+
+description: |
+ This rule checks that the `aria-` attribute specified is defined in ARIA 1.1.
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+Any attribute that starts with `aria-`.
+
+### Expectation
+
+Each target attribute which starts with `aria-` is defined in [WAI ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/).
+
+## Assumptions
+
+`aria-` attributes are used to provide assistive technologies with accessibility information. This rule does not cater to any custom `aria-` attribute that does not fall under the [WAI ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/) specification.
+
+This 
+If someone decides to make up their own `aria-` attribute and do some scripting to do other stuff with it, this rule won't account for that.
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [https://www.w3.org/TR/html-aria/#index-aria-global](https://www.w3.org/TR/html-aria/#index-aria-global)
+- [http://www.w3.org/TR/wai-aria/#states_and_properties](http://www.w3.org/TR/wai-aria/#states_and_properties)
+- [http://www.w3.org/TR/WCAG20-TECHS/G108](http://www.w3.org/TR/WCAG20-TECHS/G108)
+- [http://oaa-accessibility.org/examples/](http://oaa-accessibility.org/examples/)
+- [http://www.w3.org/TR/WCAG20/#ensure-compat-rsv](http://www.w3.org/TR/WCAG20/#ensure-compat-rsv)
+- [https://developers.google.com/web/fundamentals/accessibility/semantics-aria/](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/)
+- [WAI ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+
+## Test cases
+
+### Passed
+
+#### Pass example 1
```

Comment:
"Passed example 1" ?

Reply:
We seem to used `pass` more than `passed` and `fail` more than `failed` in the examples. Refer all the published rules. Need to agree before making any change.

Reply:
It's probably because it was in the template at some point... 
I just find it inconsistent...

Reply:
Updated to `Passed` and `Failed` accordingly.